components/console-backend-service/internal/domain/eventing/trigger/service.go

```diff
@@ -0,0 +1,187 @@
+package trigger
+
+import (
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/domain/eventing/pretty"
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/gqlschema"
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/resource"
+	notifierResource "github.com/kyma-project/kyma/components/console-backend-service/pkg/resource"
+
+	"github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/tools/cache"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+)
+
+//go:generate mockery -name=Service -output=automock -outpkg=automock -case=underscore
+//go:generate failery -name=Service -case=underscore -output disabled -outpkg disabled
+type Service interface {
+	Create(trigger *v1alpha1.Trigger) (*v1alpha1.Trigger, error)
+	CreateMany(triggers []*v1alpha1.Trigger) ([]*v1alpha1.Trigger, error)
+	Delete(trigger gqlschema.TriggerMetadataInput) error
+	DeleteMany(triggers []gqlschema.TriggerMetadataInput) error
+	List(namespace string, subscriber *gqlschema.SubscriberInput) ([]*v1alpha1.Trigger, error)
+	Subscribe(listener notifierResource.Listener)
+	Unsubscribe(listener notifierResource.Listener)
+
+	//For tests
+	GetInformer() cache.SharedIndexInformer
```

Comment:
You can create fake client with informer like that https://github.com/kyma-project/kyma/blob/master/components/console-backend-service/internal/domain/rafter/asset_service_test.go#L166 and you don't need this method. Please remove it.